tictactoeSpecific.md

```diff
@@ -0,0 +1,112 @@
+
+
+# TicTacToe Game Specification
+
+## About the Game:
+
+Tic-Tac-Toe, it's a game of two players, who take turns marking spaces (**X or O**), it can be any symbols)in a 3*3 grid.
+
+## Game Rules:
+
+As we said, it's a two-player game, where one player takes one symbol(**X**) and the second player takes one symbol(**O**).
+
+The first player to get 3 of her marks in a row (**up, down, across, diagonally**) is the winner.
+
+When all 9 squares are full, the game is over or draw.
+
+## Game design:
+
+In our design, it's a game between a computer and a human.
+
+We can design the grid in a 1D array or 2D array.
+
+
+**GRID DESIGN**:
+
+**1D ARRAY**
+1 | 2 | 3
```

Comment:
9 squares. Are you sure 1 | 2 | 3 are headers? Go for table without headers if required.